documentation/automations/guides/requirements.md

```diff
@@ -0,0 +1,7 @@
+# Requirements
+
+- Node v14
+  - This should be kept in sync between the GitHub Actions `node-version` and
+    the `volta.node` key in the `package.json` file.
+- Python 3.10
```

Comment:
I think these version numbers might be inaccurate. It's probably better to refer to the project-wide requirements here the same way other sections do. Here's an example for the documentation for reference: https://docs.openverse.org/meta/documentation/quickstart.html#prerequisites